azurerm/resource_arm_traffic_manager_endpoint.go

```diff
@@ -110,6 +110,46 @@ func resourceArmTrafficManagerEndpoint() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"custom_header": {
+				Type: schema.TypeList,
+				Elem: &schema.Resource{
```

Comment:
Could we move `Elem` below optional and force new for consistency with other resources?

Reply:
Sure